sonar-plugin-api/src/main/java/org/sonar/api/component/RubyComponentService.java

```diff
@@ -19,13 +19,14 @@
  */
 package org.sonar.api.component;
 
-import org.sonar.api.server.ServerSide;
-
 import javax.annotation.CheckForNull;
+import org.sonar.api.server.ServerSide;
 
 /**
  * @since 3.6
+ * @deprecated since 5.6
  */
+@Deprecated
 @ServerSide
 public interface RubyComponentService {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1133)